scripts/sync-workflow.js

```diff
@@ -58,3 +61,59 @@ function login(resolve, reject) {
     return resolve(resBody);
   });
 }
+
+function emptySync(loginResult) {
+  return function(resolve, reject) {
+    lr.actStart('EMPTY SYNC');
+
+    const myRequest = request.defaults({
+      headers: requestBodyUtils.getSyncRequestHeaders({
+        'X-FH-cuid': clientIdentifier,
+        'X-FH-sessionToken': loginResult.sessionToken
+      })
+    });
+
+    const reqBody = requestBodyUtils.getSyncRequestBody({
```

Comment:
Most of the stuff here I got from inspecting requests sent from the browser. Maybe some of it is not needed.